src/test/java/org/apache/commons/daemon/ProcrunDaemon.java

```diff
@@ -77,6 +77,18 @@ public static void ProcessClient(Socket clientSocket) {
                         System.out.println(ex);
                     }
                     System.exit(0);
+                } else if (buf[0] == '5') {
+                    System.out.println("5");
+                    PrintStream myout = System.out;
+                    System.setOut(originalStdout);
+                    System.out.println("Using System.out");
+                    System.out.println("5");
+                    System.out.println("Back to redirection");
+                    System.setOut(myout);
```

Comment:
Can you expand on what is going on here? I'm not following and I am wondering why this is more complex than stderr - I'd expect them to be the same.

Reply:
Because the stdout is redirected to another file in ProcrunDaemon:
https://github.com/apache/commons-daemon/blob/05cf1a47c4974292b613b959b391586d284823e5/src/test/java/org/apache/commons/daemon/ProcrunDaemon.java#L152

Reply:
done.